wsmaster/che-core-api-installer/src/main/java/org/eclipse/che/api/installer/server/InstallerRegistry.java

```diff
@@ -109,10 +109,13 @@
 
     /**
      * Traverses dependencies of all listed installers and
-     * returns properly ordered list of non-duplicated installer descriptions
+     * returns properly ordered list of non-duplicated installer descriptions.
      *
-     * @param installers
-     *         installers to fetch dependencies and order
+     * If any of {@code installerKeys} contains only id then the latest version of this installer
```

Comment:
FYI empty lines in javadocs are ignored. If you want to put a brake usually `<p/>` is used.